streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDbWindowBytesStoreSupplier.java

```diff
@@ -28,14 +28,15 @@
     private final long windowSize;
     private final boolean retainDuplicates;
 
-    @SuppressWarnings("deprecation")
+    private static final int MIN_SEGMENTS = 2;
```

Comment:
Should the `MIN_SEGMENTS` here be 3 as well or should the one above change?

Reply:
`RocksDBWindowStoreSupplier.MIN_SEMENTS` is `2` -- note that `MIN_SEGMENTS` and `NUM_SEGMENTS` are two different things.